Subject: TOMOYO: Temporarily disable in preparation for a series of changes.

For ease of review, subsequent patches are split up in a way where runtime
errors occur. Due to changes in policy syntax, it is difficult to prepare
userspace programs for "git bisect" until a series of changes has been applied.

For ease of "git bisect" for problems unrelated with TOMOYO, this patch
temporarily disables TOMOYO. You can estimate how TOMOYO will look like by
browsing http://tomoyo.sourceforge.jp/cgi-bin/lxr/source/security/ccsecurity/ .

This commit will be reverted when all changes are done.

Signed-off-by: Tetsuo Handa <penguin-kernel@I-love.SAKURA.ne.jp>
---
 security/tomoyo/tomoyo.c |    3 +++
 1 file changed, 3 insertions(+)

--- security-testing-2.6.orig/security/tomoyo/tomoyo.c
+++ security-testing-2.6/security/tomoyo/tomoyo.c
@@ -276,6 +276,9 @@ static int __init tomoyo_init(void)
 
 	if (!security_module_enable(&tomoyo_security_ops))
 		return 0;
+	printk(KERN_INFO "TOMOYO Linux is temporarily disabled due to large "
+	       "and imcompatible changes are in progress.\n");
+	return 0;
 	/* register ourselves with the security framework */
 	if (register_security(&tomoyo_security_ops) ||
 	    init_srcu_struct(&tomoyo_ss))
